test/libsolidity/util/TestFileParser.cpp

```diff
@@ -427,6 +438,34 @@ Parameter TestFileParser::parseParameter()
 		parameter.rawString += formatToken(Token::RParen);
 	}
 
+	if (boost::starts_with(_type, "fixed"))
```

Comment:
Can we simplify this so that it does not need the name of the parameter? Is it possible to accept the decimal point just about anywhere and parse `1.14` as `114` and then when re-formatting the number (where I think we know the type) put the decimal point at the right place (and add padding zeros to the right if needed)?